resources/assets/less/status.less

```diff
@@ -0,0 +1,69 @@
+/**
+*    Copyright 2015 ppy Pty. Ltd.
+*
+*    This file is part of osu!web. osu!web is distributed with the hope of
+*    attracting more community contributions to the core ecosystem of osu!.
+*
+*    osu!web is free software: you can redistribute it and/or modify
+*    it under the terms of the Affero GNU General Public License version 3
+*    as published by the Free Software Foundation.
+*
+*    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+*    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+*    See the GNU Affero General Public License for more details.
+*
+*    You should have received a copy of the GNU Affero General Public License
+*    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+*
+*/
+@import "app";
+
+.status {
+  &__header {
+    .at2x-simple("/images/headers/status.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: left center;
+    height: 130px;
+  
+    margin-top: 20px;
+    margin-bottom: 20px;
+    &:extend(.default-box-shadow);
+  }
+
+  &__header-text {
+    padding-left: 130px;
+
+    h1 {
+      padding-top: 40px;

```

Comment:
this doesn't actually center it. Try using a combination of `display: flex` and `align-items: center`.